Sensor Analysis/sensoranalysis.py

```diff
@@ -163,41 +132,44 @@ def main():
-            count = count + 1
+        if key not in notGraphedData:
+            count += 1
             if (count2 <= 7):
-                std_plot, time_plot, data_plot = stddev_plot(arr_df[:,7],dict_sensor[key],t1,t2)
+                std_plot, time_plot, data_plot = stddev_plot(sensorData["lowG_data.timeStamp_lowG"],sensorData[key],t1,t2)
             if (8 <= count2 <= 12):
-                std_plot, time_plot, data_plot = stddev_plot(arr_df[:,12],dict_sensor[key],t1,t2)
-            # if (13 <= count2 <= 20):
-            #     std_plot, time_plot, data_plot = stddev_plot(arr_df[:,20],dict_sensor[key],t1,t2)
+                std_plot, time_plot, data_plot = stddev_plot(sensorData["highG_data.timeStamp_highG"],sensorData[key],t1,t2)
+
             if (13 <= count2 <= 17):
-                std_plot, time_plot, data_plot = stddev_plot(arr_df[:,25],dict_sensor[key],t1,t2)
-            # if (26 <= count2 <= 33):
-            #     std_plot, time_plot, data_plot = stddev_plot(arr_df[:,37],dict_sensor[key],t1,t2)
-            # if (34 <= count2 <= 36):
-            #     std_plot, time_plot, data_plot = stddev_plot(arr_df[:,40],dict_sensor[key],t1,t2)
-            # if (37 <= count2 <= 39):
-            #     std_plot, time_plot, data_plot = stddev_plot(arr_df[:,43],dict_sensor[key],t1,t2)
+                std_plot, time_plot, data_plot = stddev_plot(sensorData["barometer_data.timeStamp_barometer"],sensorData[key],t1,t2)
+
             if (18 <= count2 <= 20):
-                std_plot, time_plot, data_plot = stddev_plot(arr_df[:,46],dict_sensor[key],t1,t2)
+                std_plot, time_plot, data_plot = stddev_plot(sensorData["voltage_data.timestamp"],sensorData[key],t1,t2)
             if (21 <= count2 <= 34):
-                std_plot, time_plot, data_plot = stddev_plot(arr_df[:,60],dict_sensor[key],t1,t2)
+                std_plot, time_plot, data_plot = stddev_plot(sensorData["orientation_data.timeStamp_orientation"],sensorData[key],t1,t2)
             if (35 <= count2 <= 39):
-                std_plot, time_plot, data_plot = stddev_plot(arr_df[:,65],dict_sensor[key],t1,t2)
+                std_plot, time_plot, data_plot = stddev_plot(sensorData["magnetometer_data.timestamp"],sensorData[key],t1,t2)
             if (40 <= count2 <= 45):
-                std_plot, time_plot, data_plot = stddev_plot(arr_df[:,71],dict_sensor[key],t1,t2)
+                std_plot, time_plot, data_plot = stddev_plot(sensorData["gas_data.timestamp"],sensorData[key],t1,t2)
             # fig = go.Figure()
```

Comment:
uncomment this and see what happens. it should graph some of the outputs so you can verify how yours functions vs. the original